conf.d/health_alarm_notify.conf

```diff
@@ -332,6 +332,25 @@ SLACK_WEBHOOK_URL=""
 # roles):
 DEFAULT_RECIPIENT_SLACK=""
 
+#------------------------------------------------------------------------------
+# rocketchat (rocket.chat) global notification options
+
+# multiple recipients can be given like this:
+#                  "CHANNEL1 CHANNEL2 ..."
+
+# enable/disable sending rocketchat notifications
+SEND_ROCKETCHAT="YES"
```

Comment:
This should probably default to being off like everything else does

Reply:
I took example on the SEND_SLACK variable in the same file:
https://github.com/firehol/netdata/blob/master/conf.d/health_alarm_notify.conf#L322

It's also YES:
`SEND_SLACK="YES"`

Or am I missing a point?

Reply:
Huh, I actually hadn't seen that slack notifications default to on.  That should probably be changed.

The big thing here though is that the check to see if a notification type is enabled happens very early in `alarm-notify.sh`, so turning something off by setting `SEND_*="NO"` avoids almost _all_ processing relating to that notification mechanism, while doing it by leaving the recipients empty has to go through a lot more processing before it decides to do nothing.  IOW, having something disabled with the `SEND_*` variables improves efficiency over disabling it with empty recipients.

Reply:
OK, NVM, I had forgotten that I had updated all my systems so that all the `SEND_*` variables are set to NO except the ones I want (for the above mentioned performance reasons.

GIven that what you have matches everything else (except the syslog one, but that one really does need to default to being off), you can just ignore my above comments.